src/Tests/Tests.ValueArgument.cs

```diff
@@ -37,6 +40,42 @@ public CommandLineParser.CommandLineParser InitValueArgument()
             return commandLineParser;
         }
 
+        [Fact]
+        public void ValueArgumentWithOptionalValue_shouldReturnDefaultValue_whenParameterNotPassed_andForcedDefaultValueNotSet()
+        {
+            // ARRANGE 
+            string[] args = { " " };
+            var commandLineParser = InitValueArgument();
+            // ACT 
+            commandLineParser.ParseCommandLine(args);
+            // ASSERT
+            Assert.Equal(9, valueArgumentTarget.Duration);
```

Comment:
This is the only test which fails on old code. Can you confirm that ?

Reply:
Yes indeed. This is the case where the value is initialized with the Init() method.
Other tests have their values initialized with the Parse() method because the parameter is passed.